components/Carbon.js

```diff
@@ -106,6 +106,18 @@ class Carbon extends PureComponent {
         </style>
       </div>
     )
+    let watermark = <Watermark />
+    if (this.props.customWatermark) {
+      let watermarkSource = this.props.customWatermark
+      watermark = (
+        <img
+          className="watermark"
+          style={{ height: '75px', width: '75px' }}
+          src={watermarkSource.customWatermark}
```

Comment:
You have `this.props.customWatermark.customWatermark.` . . . are you able to flatten this? (: